pkgs/applications/misc/pgmodeler/default.nix

```diff
@@ -23,7 +30,6 @@ mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "A database modeling tool for PostgreSQL";
-    longDescription = ''pgModeler (PostgreSQL Database Modeler) is an open source database modeling tool designed for PostgreSQL.'';
```

Comment:
Why did you remove the long description?

Reply:
It seemed a bit redundant to me (rephrased description).

Reply:
I've reverted it, let me know if you changed your mind.

Reply:
On a second look it is almost the same. keep it.

Reply:
Ok, I've force pushed the change.